### DISCUSSION TITLE
Spreadsheets: Locale support for date formats

### DISCUSSION THREAD

**Discussion by jimbobbly** [standardnotes/forum#2601] [Suggestions]
Is there any way to get date formats in spreadsheet cells such as 2019/09/09 or 9/9/2019?

The only ones available to select seem to be US style MM/DD/YYYY and similar, which are unreadable to most of the world.

**Comment by atmoio** [comment]
We'd need to implement locale support. This is something planned for the future but no definite date yet.

**Comment by leonahess** [comment]
Is there an update or a timeline on when this can be expected to be implemented?
